implementations/rust/ockam/ockam_command/tests/bats/nodes.bats

```diff
@@ -92,6 +92,18 @@ force_kill_node() {
   run_success "$OCKAM" node create "$n"
 }
 
+@test "node - fail to create background node when not existing identity is passed" {
+  n="$(random_str)"
+  i=$(random_str)
+  run_failure "$OCKAM" node create "$n" --identity "$i"
+}
+
+@test "node - fail to create foreground node when not existing identity is passed" {
+  n="$(random_str)"
+  i=$(random_str)
+  run_failure "$OCKAM" node create "$n" -f --identity "$i"
+}
```

Comment:
Maybe we could merge both tests into one, and also remove the optional node name argument.

Reply:
sure, done